x-pack/plugins/apm/server/routes/services/get_service_metadata_details.ts

```diff
@@ -93,7 +93,6 @@ export async function getServiceMetadataDetails({
     apm: {
       events: [
         getProcessorEventForTransactions(searchAggregatedTransactions),
-        ProcessorEvent.error,
```

Comment:
Why should this be removed? Do we not also need metadata for when a service only has error events in the given time range?

Reply:
In order to match the node instances table which doesn't include error events in the query.  

even if the service has only error events in the given range we can't rely on the metrics to display the metadata?

Reply:
@kpatticha I'm not sure if I understand your question. Why do we have more service nodes for error events than metric events? Is that an issue with opbeans data? In what kind of real world scenarios would that happen?

Reply:
it seems there is an issue with opbeans data only for `opbeans-go` service.  The other instances look okay. 
However, I wasn't able to understand how this did happen. 

Closing the issue and reopen it if it needed